src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
I meant that as it is, for instance, we need different overrides for different languages depending on the quote delimiter used for their string literals. Can we do something more intelligent with the override parsing?